sdk/src/main/kotlin/com/processout/sdk/core/logger/POLogger.kt

```diff
@@ -1,49 +1,61 @@
 package com.processout.sdk.core.logger
 
-internal class POLogger private constructor() {
+import com.processout.sdk.core.annotation.ProcessOutInternalApi
+
+@ProcessOutInternalApi
+class POLogger private constructor() {
 
     companion object {
+        internal const val ATTRIBUTE_LINE = "Line"
+        internal const val ATTRIBUTE_INVOICE_ID = "InvoiceId"
+        internal const val ATTRIBUTE_GATEWAY_CONFIGURATION_ID = "GatewayConfigurationId"
+
         @Volatile
-        private var services = emptyArray<LoggerService>()
-        private val servicesLock = mutableListOf<LoggerService>()
+        private var services = emptyArray<POLoggerService>()
+        private val servicesLock = mutableListOf<POLoggerService>()
 
-        fun add(service: LoggerService) {
+        @ProcessOutInternalApi
```

Comment:
just curious, do we need to mark methods with `ProcessOutInternalApi when `POLogger` is already marked with same annotation?

Reply:
Removed unnecessary ProcessOutInternalApi annotations on logger methods. It will notify.